pipelines/build/common/create_job_from_template.groovy

```diff
@@ -73,6 +73,7 @@ pipelineJob("$buildFolder/$JOB_NAME") {
                 <dt><strong>BUILD_ARGS</strong></dt><dd>args to pass to makejdk-any-platform.sh</dd>
                 <dt><strong>NODE_LABEL</strong></dt><dd>Labels of node to build on</dd>
                 <dt><strong>ADDITIONAL_TEST_LABEL</strong></dt><dd>Additional label for test jobs</dd>
+                <dt><strong>KEEP_TEST_REPORTDIR</strong></dt><dd>If true, test report dir will be kept even when the testcase passes, failed testcases always keep the report dir. Does not apply to JUnit jobs which are always kept, eg.openjdk.</dd>
```

Comment:
Would be worth adding why we are doing this, e.g. OpenJ9 report artifacts are BIG :-)

Reply:
yes good idea